### DISCUSSION TITLE
Z2T: Multipliers in attribute synonyms

### DISCUSSION THREAD

**Discussion by SampusCZE** [arendst/Tasmota#17138] [Q&A] [answered]
Hi,
im trying to make zb plugins for my Tuya devices, but i'm not able to make multipliers works.

**Example - wall dimmer:**
`EF00/0202=0008/0000,0`
Dimmer value output: 0-1000
`EF00/0202=0008/0000,10`
Dimmer value output: 0-1000

It looks like either multipliers dont works or some explicit type mismatch in Tuya vs. ZCL

Any ideas?

**Comment by s-hadinger** [comment, marked as answer]
I am currently writing the doc for it. But it should be something like:
 `EF00/0202=0008/0000,mul:10`

**Reply by s-hadinger** [reply]
More precisely, changing from 0..1000 to 0..254 needs the following:

`EF00/0202=0008/0000,mul:254,div:1000`

I may need to add something like "change from range A..B to C..D"

**Reply by SampusCZE** [reply]
Working nicely. Thanks @s-hadinger 

Is there any way to disable multiplier when sending(writing) command? 
Reading working nicely acording to multipliers and from 1000 im getting 100, but sending 100 converts to 10 on output.

**Reply by s-hadinger** [reply]
Can you please send the logs with `weblog 3`. Multipliers are supposed to do the opposite work when sending. It looks like there is a bug

**Reply by SampusCZE** [reply]
My bad, error was in plugin. New doc helped! :) 

Thank you
